package.json

```diff
@@ -36,7 +36,7 @@
     "@turf/intersect": "^6.5.0",
     "@types/react": "~18.2.14",
     "aws-amplify": "~6.0.6",
-    "axios": "~1.6.0",
+    "axios": "^1.6.7",
```

Comment:
Is there reason to switch from `~` to `^`?
If not, we agreed to keep minor versions locked.